browser/app/js/objects/ObjectsBulkActions.js

```diff
@@ -107,40 +130,39 @@ export class ObjectsBulkActions extends React.Component {
             }
             disabled={!checkedObjectsCount}
           >
-            <img src={iconDelete} alt="" />
+            {icons.trash}
             <span>Delete selected</span>
           </button>
-
-          {loggedIn && (
-            <button
-              id="share-checked"
-              className="object-actions__item"
-              onClick={this.shareObject.bind(this)}
-              disabled={checkedObjectsCount != 1}
-            >
-              <img src={iconShare} alt="" />
-              <span>Share</span>
-            </button>
-          )}
           <button
             id="download-checked"
             className="object-actions__item"
             onClick={this.handleDownload.bind(this)}
             disabled={!checkedObjectsCount}
           >
-            <img src={iconDownload} alt="" />
+            {icons.download}
```

Comment:
As we are not using `iconDownload`, `iconShare`, `iconDelete` and `iconClose`, I think we can get rid of the imports on lines 32-35